artiq/examples/kasli_basic/repository/kasli_tester.py

```diff
@@ -16,11 +18,18 @@ def chunker(seq, size):
 
 
 def is_enter_pressed() -> TBool:
-    if select.select([sys.stdin,], [], [], 0.0)[0]:
-        sys.stdin.read(1)
-        return True
-    else:
-        return False
+    if sys.platform == 'linux2':
+        if select.select([sys.stdin,], [], [], 0.0)[0]:
+            sys.stdin.read(1)
+            return True
+        else:
+            return False        
+    elif sys.platform == 'win32':   
+        if msvcrt.kbhit():
+            sys.stdin.read(1)
+            return True
+        else:
+            return False  
```

Comment:
The function will break if ``sys.platform`` is neither ``linux2`` or ``win32``.
I would simply test for ``os.name == "nt"`` and use ``msvcrt`` in that case and ``select`` (which is available on more platforms) in all other cases.